src/gui/main-styles.js

```diff
@@ -574,6 +574,14 @@ styles.registerStyle('main', () => {
 			'margin-top': requiresStatusBarHack() ? "20px" : 'env(safe-area-inset-top)' // insets for iPhone X)
 		},
 
+		'.suggestions': {
```

Comment:
We don't do class per component, we do class per use case. We can compose this out of other classes we have.

Reply:
I changed it.